tests/queries/0_stateless/01744_fuse_sum_count_aggregate.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS fuse_tbl;
+CREATE TABLE fuse_tbl(a Int8, b Int8) Engine = Log;
+INSERT INTO fuse_tbl SELECT number, number + 1 FROM numbers(1, 20);
```

Comment:
I think we need a performance test as well.

Reply:
How can I add a performace test, any advice?

Reply:
Just copy some existing one like this:
https://github.com/ClickHouse/ClickHouse/blob/master/tests/performance/avg_weighted.xml

You probably don't need a special table and can calculate average over `numbers`.

The test queries should run between 0.1 and 1 seconds to give stable results. You can test it locally as described here: https://github.com/ClickHouse/ClickHouse/tree/master/docker/test/performance-comparison#run-a-single-test-on-the-already-configured-servers